src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -79,8 +79,10 @@ uct_cuda_ipc_iface_is_reachable_v2(const uct_iface_h tl_iface,
                                    const uct_iface_is_reachable_params_t *params)
 {
     return uct_iface_is_reachable_params_addrs_valid(params) &&
+#if !HAVE_CUDA_FABRIC
```

Comment:
maybe we can introduce an env var which would manage MNNVL enablement (i.e. make devices on different nodes unreachable if it is disabled)